.jenkins/pytorch/build.sh

```diff
@@ -30,10 +30,16 @@ cmake --version
 pip install -r requirements.txt || true
 
 if [[ "$BUILD_ENVIRONMENT" == *rocm* ]]; then
+  # This is necessary in order to cross compile (or else we'll have missing GPU device).
   export HCC_AMDGPU_TARGET=gfx900
+
+  # These environment variables are necessary in order to avoid errors in HIP utility scripts.
```

Comment:
unclear what the underlying issue is. is this a HIP shortcoming? what scripts? what errors?

Reply:
These environment variables are not set on CI when we were running as the Jenkins user. The HIP Utility scripts require these environment variables to be set in order to run without error.